config/webpack.config.js

```diff
@@ -96,22 +96,40 @@ const supportedLanguagesRE = new RegExp(
   `/(${supportedLanguages().join('|')})$`
 );
 
-const commonConfig = function() {
+const commonConfig = function(buildName = 'core') {
   var production =
     process.env.RAILS_ENV === 'production' ||
     process.env.NODE_ENV === 'production';
   const mode = production ? 'production' : 'development';
   const config = {};
+
+  if (!production) {
```

Comment:
There already is an if/else on production. Should this move in the `else` block there?